common/src/main/java/com/microsoft/identity/common/internal/authscheme/PopAuthenticationSchemeInternal.java

```diff
@@ -68,26 +70,70 @@
     @SerializedName(NONCE)
     private String mNonce;
 
+    @SerializedName(CLIENT_CLAIMS)
+    private String mClientClaims;
+
     /**
      * Constructor for gson use.
      */
     PopAuthenticationSchemeInternal() {
         super(SCHEME_POP);
     }
 
+    @Deprecated
     public PopAuthenticationSchemeInternal(@NonNull final IClockSkewManager clockSkewManager,
-                                    @Nullable final String httpMethod,
-                                    @NonNull final URL url,
-                                    @Nullable final String nonce) {
+                                           @Nullable final String httpMethod,
+                                           @NonNull final URL url,
+                                           @Nullable final String nonce) {
         super(SCHEME_POP);
         mClockSkewManager = clockSkewManager;
         mHttpMethod = httpMethod;
         mUrl = url;
         mNonce = nonce;
     }
 
+    public PopAuthenticationSchemeInternal(@NonNull final IClockSkewManager clockSkewManager,
```

Comment:
Increment min broker hello api

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1208/commits/c30c347cda62d33ef29a61e18e5a75504e45ce65